File: learn-C#-w3c/classes/classes2.cs
using System;

namespace MyApplication
{
    class Car
    {
        public string model;
        public string color;
        public int year;

        // Create constructor
        public Car(string modelname, string modelcolor, int modelyear)
        {
            model = modelname;
            color = modelcolor;
            year = modelyear;
        }

        static void Main(string[] args)
        {
            Car Ford = new Car("Mustang", "Red", 1969);
            Console.WriteLine($"{Ford.color} {Ford.year} {Ford.model}");
        }
    }
}

File: Tutorials/Programming/learn_C++_w3c/classes2.cpp
#include <iostream>
#include <string>
using namespace std;

class Car {
    public:
    string brand;
    string model;
    int year;

    // Constructor
    Car(string x, string y, int z) {
        brand = x;
        model = y;
        year = z;
    }
};

int main() {
    Car carObj1("BMw", "X5", 1999);
    cout << carObj1.brand << endl;
    cout << carObj1.model << endl;
    cout << carObj1.year;
    return 0;

}